docs/logs.md

```diff
@@ -34,3 +34,10 @@ You can get execution logs using one of the following methods:
 - Get the logs using [Tekton Dashboard](https://github.com/tektoncd/dashboard).
 
 - Configure an external service to consume and display the logs. For example, [ElasticSearch, Beats, and Kibana](https://github.com/mgreau/tekton-pipelines-elastic-tutorials).
+
+
+# Logs Credential Filter
+
+Often it is the case that secret values will sneak into the ouput logs of your pipelines. You can either handle this manually by avoiding to print them in the output or use the feature for filtering secret values from the output log. To enable this feature set the `enable-logging-credentials-filter` value in the `feature-flags` config map to `"true"` (see [Customizing the Pipelines Controller behavior](install.md#customizing-the-pipelines-controller-behavior)).
+
+When the credentials filter is active the tekton entrypoint will redact the secrets found in environment variables and mounted files from secrets. It will replace them with `[REDACTED:<secret-name>]` where secret name is either the environment variable or the file name that was redacted.
```

Comment:
Does it make sense to add the following [design after discussions](https://github.com/tektoncd/pipeline/pull/4837#issuecomment-1142377168) to the documentations?
```
This design does not transmit actual secrets. Only the places where the pod can find secrets it has attached to it. And those are the secrets that need to be redacted. Because only those can be used in the corresponding pipeline step.
```

Reply:
Makes sense, I will add that.